src/ReactNativeZoomableView.tsx

```diff
@@ -349,11 +349,28 @@ class ReactNativeZoomableView extends Component<ReactNativeZoomableViewProps, Re
         const currentElementWidth = originalWidth * changeObj.zoomLevel;
         const currentElementHeight = originalHeight * changeObj.zoomLevel;
 
+        // gentle reminder for the following logic:
+        //    RATIO = W / H
+        //    H = W / RATIO
+        //    W = H * RATIO
+
+        const aspectRatioOnMount = originalHeight && originalWidth ? originalWidth / originalHeight : 1;
+        const { contentAspectRatio = aspectRatioOnMount } = this.props;
+
+        const wasWidthCroppedOnMount = aspectRatioOnMount < contentAspectRatio;
```

Comment:
Could you please add a small comment explaining the logic here? It seems obvious, but up until now we kind of follow the approach to summarize the following block of logic, to make it easier to get into. :)